### ISSUE TITLE
Strange behavior for saturation over field extension

### ISSUE THREAD

**Issue by aseceleanu** [Macaulay2/M2#779]
The following example has been pointed out to me recently by Brian Harbourne.

We construct an ideal I defining 16 reduced points in P^2 over a field F containing the square root of 5 (see the specific ideal below). The saturation of I^3 with respect to the maximal ideal m=(x,y,z) should be a degree 96=16*6 scheme. However, the command saturate(I^3) produces a degree 22 scheme and the same result is obtained by saturate(I^3,m). Iteratively taking colon ideals  I^3:m^i for i>0 until this stabilizes produces the correct result.

My questions are:
1) What goes wrong with saturate(I^3)?
2) How does Macaulay 2 actually compute saturation in this case? The documentation seems to indicate that saturation is computed by taking iterated colon ideals, but the result above shows that is probably not the case.

Here is the sample computation:
```
n=2;
K=toField(QQ[t]/(t^2-5));
R=K[x_0..x_n];
Pts={{1,1,1},
{1,-1,1},
{-1,1,1},
{-1,-1,1},
{2+t,2+t,1},
{2+t,-(2+t),1},
{-(2+t),2+t,1},
{-(2+t),-(2+t),1},
{1,2+t,1},
{-1,-(2+t),1},
{2+t,1,1},
{-(2+t),-1,1},
{2,1+t,0},
{1+t,2,0},
{0,1,0},
{1,0,0}};
#Pts
-- This gives 16
Ids=apply(Pts,s->(trim minors(2,vars(R)||matrix{s})));
I=intersect(Ids);
degree I
-- This gives 16 as expected
J=saturate(I^3);
degree J
-- This gives 22 !?
-- one way to compute the correct saturation in this case is the following
I3=ideal(1_R);
apply(Ids,s->(I3=trim intersect(I3,s^3)));
degree I3
-- This gives 16*6=96 as expected
-- another way to compute saturation is by repeated coloning
i=1;
L=I^3:m;
while L!=(I^3:(m^(i+1))) do ( i=i+1; L=I^3:m^i;)
degree L
-- this time we get 96
L== I3
-- true so L is the right ideal
```

**Comment by DanGrayson** [comment]
Thanks!

**Comment by mahrud** [comment]
fixed in #1385
